_projects/2024/100546612/100546612.Rmd

```diff
@@ -0,0 +1,1007 @@
+
+[OurWorldinData](https://ourworldindata.org/extreme-poverty-in-brief#article-citation) takes a closer look at the current developments in poverty. Particularly, they are interested in highlighting the necessity to differentiate between people living in poverty, and people living in *extreme poverty*, i.e. living on less than 2.15\$ a day, as without this distinction we would not understand the crucial differences and trends in living standards within impoverished people. In the graph below we can observe how with this metric relates to the GDP of a country. Tackling out previously mentioned key points, we see the important share of wealth that has propelled many world regions out of extreme poverty in the last 200 years, and at the same time recognize which world regions have disproportionate percentages of extreme poverty in comparison to other regions. While it takes just a bit to understand the underlying relationship that OurWorldinData seeks to visualize, the point is that economic growth implies poverty decline, and stagnated economies imply higher poverty rates.
+
+![Original graph sourced from [Our World in Data](https://ourworldindata.org/extreme-poverty-in-brief#article-citation).](Growth-and-poverty-since-1820.png){.external width="100%"}
+
+Hence, to quote Max Roser, the author of this graph and corresponding article, "what needs explanation is not poverty, but prosperity." I find this point particularly intriguing, and will refer back to it in my proposal for the graph improvement. However, first let us take a look at the replication of the original
+
+### The original graph
+
+The original graph portrays the relationship between the share living in extreme poverty and the GDP per capita in 8 world regions and the global average between the years 1817 and 2017.
+
+::: {style="display: flex; align-items: flex-start; gap: 15px; overflow: hidden;"}
+```{=html}
+<img src="Growth-and-poverty-since-1820.png" 
+     style="width: 300px;"/>
+```
+<p>The graphs strengths include its clear comparison between the world regions wealth and poverty trends, which highlight the significant variation between regions, and its historical perspective, which provides some context for the global and regional progress. Visually, the regional facets and colouring contribute to their acute distinction and provide clarity and readability.</p>
+:::
+
+## Before we start...
+
+### Library and fonts
+
+To replicate the graph, the below packages and font families were used. The fonts were retrieved, for free, from Google Fonts. The libraries are commented within the chunks they started being used.
+
+```{r library}
+
+library(readxl) 
+library(tidyverse) 
+library(dplyr) 
+library(ggplot2)
+library(forcats)
+library(ggtext)
+library(grid)
+
+# dowload fonts
+sysfonts::font_add_google("Playfair Display", family = "playfair display")
+sysfonts::font_add_google("Lato", family = "lato")
+sysfonts::font_add(regular = "./Lato/Lato-Light.ttf", family = "lato_light")
```

Comment:
It's better to load the font again from Google Fonts with the appropriate weight. Remove please the folder with the font sources, and here you can do something like:

```r
sysfonts::font_add_google("Lato", family = "lato_light", regular.wt=300)
```

Or less if you need it thinner.